lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+
+from matplotlib.compat.subprocess import Popen
+import pytest
+
+
+# Minimal smoke-testing of the backends for which the dependencies are
+# PyPI-installable on Travis.  They are not available for all tested Python
+# versions so we don't fail on missing backends.
+#
+# Once the Travis build environment switches to Ubuntu 14.04, we should be able
+# to add wxagg (which has wheels for 14.04 but not for 12.04).
+
+
+def _get_available_backends():
+    backends = []
+    for module_name, backend in [
+            ("PyQt5", "qt5agg"),
+            ("Tkinter", "tkagg"),
+            ("tkinter", "tkagg")]:
+        try:
+            __import__(module_name)
```

Comment:
https://docs.python.org/3.6/library/functions.html#__import__ <- This suggests to use import lib instead, is there a reason not to here?  I have a vague sense of this playing badly with pypi.

Importing multiple GUI frameworks into one process can still cause problems because some of them install something onto `PyOS_InputHook`, but given that within the context of the test suite we should never land in the repl I think this is ok.

https://docs.python.org/3.6/library/importlib.html#importlib.util.find_spec (for py3) and https://docs.python.org/2.7/library/imp.html#imp.find_module (for py2) might be a better option.